### PR TITLE
2.1 beta downloads

### PR DESCRIPTION
**Pull request by Be-ing**

### CODE REVIEW

**Comment by Be-ing** [comment]
I'm confused why GitHub shows this has conflicts when it merges fine locally...

File: pages/download.html [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
-unstable_win32_analytics_conversion: /downloads/2.0.0-rc2-win32
-unstable_win64_analytics_conversion: /downloads/2.0.0-rc2-win64
-unstable_osxintel_analytics_conversion: /downloads/2.0.0-rc2-osxintel
-unstable_ubuntu32_analytics_conversion: /downloads/2.0.0-rc2-ubuntu-trusty32
-unstable_ubuntu64_analytics_conversion: /downloads/2.0.0-rc2-ubuntu-trusty64
-unstable_ubuntu_ppa_analytics_conversion: NOT YET EXTRACTED, FIX ME BELOW
-unstable_linuxsrc_analytics_conversion: /downloads/2.0.0-rc2-linuxsrc
+unstable_short_version: Mixxx 2.1
+unstable_long_version: Mixxx 2.1 Beta
+unstable_builds: http://downloads.mixxx.org/builds/2.1/release/
```
Comment (rryan) [inline pages/download.html]: https?

File: pages/download.html [hunk 2]
```diff
@@ -124,35 +126,29 @@ <h2><img src="{% static '/static/images/download_ubuntu.png' %}" class="feature_
...
-      {% with "<a href='https://launchpad.net/~mixxx/+archive/mixxxbetas' onclick=\"javascript:trackDownload('/downloads/2.0.0-rc2-ubuntu-ppa');\">"|add:mixxx_ppa|add:"</a>" as ppa_link %}
+      {% with "<a href='https://launchpad.net/~mixxx/+archive/mixxxbetas' onclick=\"javascript:trackDownload('/downloads/2.1.0-ubuntu-ppa');\">"|add:mixxx_ppa|add:"</a>" as ppa_link %}
       {% blocktrans %}This will install the latest version of Mixxx from the {{ ppa_link }} on Launchpad.{% endblocktrans %}
       {% endwith %}
     </p>
 
-    <p><a href="{{ unstable_ubuntu32 }}"
-          onclick="javascript:trackDownload('{{ unstable_ubuntu32_analytics_conversion }}');">
-        {% blocktrans %}{{ unstable_long_version }} for 32-bit Ubuntu Trusty (and newer){% endblocktrans %}</a></p>
+    <p>{% blocktrans %}New installers will be made available every time the code is changed. If you are using the PPA, these will be installed automatically with apt-get. Otherwise, you can download individual packages to install and check back periodically for updates until Mixxx 2.1 is released:{% endblocktrans %}</p>
```
Comment (rryan) [inline pages/download.html:134]: s/installers/packages/ ?

**Comment by rryan** [comment]
Is it possible you don't have https://github.com/mixxxdj/website/commit/b0377479c54f2d414b00536662045bc1469344fb in your local 'website' branch? That's what looks conflicted using GitHub's "show conflicts" view.

**Comment by rryan** [comment]
content LGTM

**Comment by rryan** [comment]
Want to update this to use the latest links?

**Comment by Be-ing** [comment]
> Is it possible you don't have b037747 in your local 'website' branch? That's what looks conflicted using GitHub's "show conflicts" view.

Yeah that was the issue, thanks.

File: pages/download.html [hunk 3]
```diff
@@ -91,35 +83,39 @@ <h1>{% blocktrans %}Download Now ({{ unstable_short_version }} Beta){% endblockt
...
+  </div>
 
-    <h2><img src="{% static '/static/images/download_mac.png' %}" class="feature_icon">{{ mac_osx }}</h2>
+  <div class="halfbox_right darkborder macbox">
+    <h2><img src="{% static '/static/images/download_mac.png' %}" class="feature_icon" style="margin-top: -7px;">{{ mac_osx }}</h2>
 
-    <p><a href="{{ unstable_osxintel }}"
+    <p><a href="{{ unstable_builds }}"
           onclick="javascript:trackDownload('{{ unstable_osxintel_analytics_conversion }}');">
         {% blocktrans %}{{ unstable_long_version }} for Mac OS X 10.6+ (Intel){% endblocktrans %}</a></p>
```
Comment (esbrandt) [inline pages/download.html:111]: Minimum required version is macOS 10.8
https://github.com/mixxxdj/mixxx/commit/2313e72944ac6e333ffc3351db68ae9bb45d5992#diff-4c2d3d5e7c14ce76e657efbdb8d11d7e

**Pull request merged**